include/pmacc/math/complex/Complex.tpp

```diff
@@ -180,6 +180,24 @@ struct Abs2< ::pmacc::math::Complex<T_Type> >
     }
 };
 
+    /*  Specialize log() for complex numbers. */
+    template< typename T_Type >
+    struct Log< ::pmacc::math::Complex< T_Type > >
+    {
+        typedef typename ::pmacc::math::Complex< T_Type >::type result;
+        using type = T_Type;
+
+        HDINLINE result operator( )( ::pmacc::math::Complex< T_Type > const & other )
+        {
+            return pmMath::log( pmMath::abs( other ) ) +
+            ::pmacc::math::Complex< T_Type >(
```

Comment:
I have as requested indented the entire operator function within its name space. What else do want to indent?